v2/tools/generator/internal/codegen/pipeline/add_cross_resource_references.go

```diff
@@ -36,14 +36,18 @@ func AddCrossResourceReferences(configuration *config.Configuration, idFactory a
 			var isCrossResourceReferenceErrs []error
 
 			isCrossResourceReference := func(typeName astmodel.TypeName, prop *astmodel.PropertyDefinition) bool {
-				isReference, found := configuration.ARMReference(typeName, prop.PropertyName())
-				if DoesPropertyLookLikeARMReference(prop) && !found {
+				isReference, err := configuration.ARMReference(typeName, prop.PropertyName())
+				if DoesPropertyLookLikeARMReference(prop) && err != nil {
```

Comment:
I realise the old version of this had no way to return errors anyway so we know that any error is actually going to be a not-found error, but for future-proofing it might be better to have this check for a specific not-found error type rather than just not-nil. Really just so the wrapped context won't be misleading if a different error case is introduced somewhere underneath `configuration.ARMReference`.

Reply:
Done.